UI/cypress/e2e/location-aware-results.feature

```diff
@@ -15,3 +15,15 @@ Feature: Tuition Partner provides online but not in-school tuition in an area
     When they select 'any' tuition type
     And they select the tuition partner 'Action Tutoring'
     Then they see the tuition types 'In School, Online'
+
+  Scenario: Only shows cost for tuition type available in the postcode’s location
+    Given a user has arrived on the 'Search results' page for 'Key stage 2 English' for postcode 'SK1 1EB'
+    When they select 'any' tuition type
```

Comment:
This when step implies (to me) that changing the tuition type is connected to the prices that are shown on the details page. I don't think it's required for any of the scenarios in the feature. Was it added to the first scenario so it had a when step then included in the others for consistency?

Reply:
Yes, you're right - the tuition type is not relevant to the test.  I have removed that step from all the scenarios.